### ISSUE TITLE
Incorrect resolving for scoped import maps

### ISSUE THREAD

**Issue by costingeana** [guybedford/es-module-shims#33]
Hello, @guybedford,
The last es-module-shims version fails to identify the scoped import maps.

This is a piece of the import map that I use (generated with [jspm](https://jspm.org/)):
```
{
  "imports": {
    "@hubgets/heros.uil.loader": "./jspm_packages/npm/@hubgets/heros.uil.loader@1.20190717.45536/lib/index.js",
    "@hubgets/heros.uil.loader/": "./jspm_packages/npm/@hubgets/heros.uil.loader@1.20190717.45536/"
},
 "scopes": {
    "jspm_packages/npm/@hubgets/heros.uil.loader@1.20190717.45536/": {
      "@hubgets/heros.datalocal": "../heros.datalocal@1.20190606.15500/lib/index.js",
      "@hubgets/heros.datalocal/": "../heros.datalocal@1.20190606.15500/",
      "@hubgets/lib.error": "../lib.error@1.20190717.49282/lib/index.js",
      "@hubgets/lib.error/": "../lib.error@1.20190717.49282/",
      "@hubgets/lib.useragent": "../lib.useragent@1.20190717.23672/lib/index.js",
      "@hubgets/lib.useragent/": "../lib.useragent@1.20190717.23672/"
    }
}
```
I investigated a little bit and I think the problem's root is in `parseImportMap`: https://github.com/guybedford/es-module-shims/blob/c2ad855e25128d4f44bfb19463ecdf79ac76b430/src/common.js#L144

I think this line should be:
```
scopes[resolvedScopeName] = resolvePackages(scope, resolvedScopeName) || {};
```
`baseUrl` for scoped packages should be the resolved url of their 'parent'.

Please let me know if you think I identified correctly the cause of the issue and I'll come up with a pull request.

Thanks!

**Comment by guybedford** [comment]
This is actually working as designed, and is a bug in jspm itself rather (and SystemJS, with the PR for SystemJS currently at https://github.com/systemjs/systemjs/pull/1975).

The import maps specification changed such that scopes are resolved relative to the import map base, instead of the scope itself.

**Comment by costingeana** [comment]
Thanks for your answer. Is there any issue added in jspm related to this matter?

**Comment by guybedford** [comment]
See https://github.com/jspm/jspm-cli/pull/2493.

**Comment by costingeana** [comment]
Thanks!

**Comment by guybedford** [comment]
Sure, this fix is now released in 2.0.0-beta.7. That should resolve this, but please let me know if you have further issues at all.

**Comment by costingeana** [comment]
I just tested it and it works. Thanks once again!
